taskcluster/tc-asserts.sh

```diff
@@ -42,10 +42,14 @@ assert_correct_inference()
   else
       echo "!! Non matching output !!"
       echo "got: <${phrase}>"
-      echo "xxd:"; echo "${phrase}" | xxd
+      if [ -x "$(command -v xxd)" ]; then
```

Comment:
we should just install `xxd` package

Reply:
It's not available in the repo for this image.

Reply:
(For Ubuntu 16.04 at least)

Reply:
it's in buster: https://packages.debian.org/search?keywords=xxd